docs/operators.md

```diff
@@ -14,9 +14,9 @@ An operator consists, broadly, of collections of runnable controllers, one type
 but a user can easily extend this by having a new controller which implements the `operator.Controller` interface.
 
 The controller offered by the SDK is the `operator.InformerController`, which is a controller that is composed of two sets of objects:
-* **Informers**, which are given a particular CRD and will notify the controller on changes, and
-* **Watchers**, which subscribe to changes for a particular CRD kind and will be notified about any changes from a relevant Informer
-Multiple Watchers can watch the same resource kind, and when a change occurs, they will be called in the order they were added to the controller.
+* **Informers**, which are given a particular CRD and will notify the controller on changes;
+* **Watchers**, which subscribe to changes for a particular CRD kind and will be notified about any changes from a relevant Informer. Multiple Watchers can watch the same resource kind, and when a change occurs, they will be called in the order they were added to the controller.;
+* **Reconcilers**, which subscribe to changes in the state of a particular CRD kind and will be noticied about any changes from a relevant Informer, its objective is to ensure that the current state of resources matches the desired state. Multiple Reconcilers can watch the same resource kind, and when a change occurs, they will be called in the order they were added to the controller.
```

Comment:
We probably should explain the difference between watchers and reconcilers and that it's enough to specify one or the other. As it stand right now one may erroneously think that it's necessary to define all three – informer, watcher and a reconciler.

Reply:
Don't you think the descriptions below (line 21 to 30) define well enough the differences between Watchers and Reconcilers? What additional information you think it's worth adding here? 
Totally agree about the need to specify users don't need both watcher and reconcilers to make things work. I added that on  line 32 👍🏼

Reply:
Ah, I've missed the descriptions due to the diff being collapsed. I think the way it is right now is good!